msprime/demography.py

```diff
@@ -1282,17 +1639,38 @@ def __post_init__(self):
         if self.destination is not None:
             self.dest = self.destination
 
-    def get_ll_representation(self, num_populations=None, demography=None):
+    def get_ll_representation(self, num_populations=None):
         # We need to keep the num_populations argument until stdpopsim 0.1 is out
         # https://github.com/tskit-dev/msprime/issues/1037
         return {
             "type": "mass_migration",
             "time": self.time,
-            "source": _convert_id(demography, self.source),
-            "dest": _convert_id(demography, self.dest),
+            "source": self._convert_id(self.source),
+            "dest": self._convert_id(self.dest),
             "proportion": self.proportion,
         }
 
+    def _assert_equivalent(
+        self,
+        other: StateChangeEvent,
+        *,
+        rel_tol: float,
+        abs_tol: float,
+    ):
+        super()._assert_equivalent(other, rel_tol=rel_tol, abs_tol=abs_tol)
```

Comment:
Is this supposed to be here? I might be missing something, but I didn't see this method in the super class hierarchy.

Reply:
No, good spot. I've gone back and forth on how to do the ``equivalent`` stuff. Will finalise today, hopefuly.